### PR TITLE
[docs-only] Deprecate some web envvars

### PR DESCRIPTION
**Pull request by mmattel**
Fixes: #6226 (Fix WEB_UI_CONFIG und WEB_UI_PATH envvars)

* The `WEB_UI_CONFIG` and `WEB_UI_PATH` envvars need deprecations and decription rewrites to make the use clear.
* Update the `web` readme

### CODE REVIEW

**Comment by mmattel** [comment]
@dragonchaser would like to get a statement from @kobergj if more changes are needed.
Setting to draft to avoid accidentially merging.

File: services/web/README.md [hunk 1]
```diff
@@ -9,6 +9,14 @@ The web service also provides a minimal API for branding functionality like chan
 
 If you want to use your custom compiled web client assets instead of the embedded ones, then you can do that by setting the `WEB_ASSET_PATH` variable to point to your compiled files. See [ownCloud Web / Getting Started](https://owncloud.dev/clients/web/getting-started/) and [ownCloud Web / Setup with oCIS](https://owncloud.dev/clients/web/backend-ocis/) for more details.
 
-## WebUI Options
+## WebUI Configuration
```
Comment (EParzefall) [inline services/web/README.md:12]:
```suggestion
## Web UI Configuration
```

File: services/web/README.md [hunk 2]
```diff
@@ -9,6 +9,14 @@ The web service also provides a minimal API for branding functionality like chan
...
 
-Beside theming, the behaviour of the embedded WebUI can be configured via options. See the environment variables `WEB_OPTION_xxx` for more details.
+Note that single configuration settings of the embedded WebUI can be defined via `WEB_OPTION_xxx` environment variables. If a json based configuration file is used via the `WEB_UI_CONFIG_FILE` environment variable, these configurations take precedence over single options set.
```
Comment (EParzefall) [inline services/web/README.md:14]:
```suggestion
Note that single configuration settings of the embedded web UI can be defined via `WEB_OPTION_xxx` environment variables. If a json based configuration file is used via the `WEB_UI_CONFIG_FILE` environment variable, these configurations take precedence over single options set.
```

File: services/web/README.md [hunk 3]
```diff
@@ -9,6 +9,14 @@ The web service also provides a minimal API for branding functionality like chan
...
+
+### WebUI Options
```
Comment (EParzefall) [inline services/web/README.md:16]:
```suggestion
### Web UI Options
```

File: services/web/README.md [hunk 4]
```diff
@@ -9,6 +9,14 @@ The web service also provides a minimal API for branding functionality like chan
...
+
+Beside theming, the behaviour of the WebUI can be configured via options. See the environment variables `WEB_OPTION_xxx` for more details.
```
Comment (EParzefall) [inline services/web/README.md:18]:
```suggestion
Beside theming, the behavior of the web UI can be configured via options. See the environment variables `WEB_OPTION_xxx` for more details.
```

File: services/web/README.md [hunk 5]
```diff
@@ -9,6 +9,14 @@ The web service also provides a minimal API for branding functionality like chan
...
+
+### WebUI Config File
```
Comment (EParzefall) [inline services/web/README.md:20]:
```suggestion
### Web UI Config File
```

File: services/web/README.md [hunk 6]
```diff
@@ -9,6 +9,14 @@ The web service also provides a minimal API for branding functionality like chan
...
+
+When defined via the `WEB_UI_CONFIG_FILE` environment variable, the configuration of the WebUI can be made with a [json based](https://github.com/owncloud/web/tree/master/config) file.
```
Comment (EParzefall) [inline services/web/README.md:22]:
```suggestion
When defined via the `WEB_UI_CONFIG_FILE` environment variable, the configuration of the web UI can be made with a [json based](https://github.com/owncloud/web/tree/master/config) file.
```

**Review by EParzefall — COMMENTED** [review]
Spelling of web UI fixed

**Review by kobergj — APPROVED** [review]

**Pull request merged**